.gitignore

```diff
@@ -13,3 +13,5 @@ coverage.html
 .glide
 .scBuildImage
 .init
+.generate*
+**/zz*.go
```

Comment:
We talked offline yesterday about checking in generated code, and I think we're in agreement that it's a good idea to check in. Can we remove this line to achieve that?